code/modules/client/preference_setup/global/02_settings.dm

```diff
@@ -66,7 +66,7 @@
 		"asfx_togs" = pref.asfx_togs,
 		"lastmotd" = pref.motd_hash,
 		"lastmemo" = pref.memo_hash,
-		"parallax_toggles" = pref.parallax_togs,
+		"parallax_toggles" = pref.toggles_secondary,
```

Comment:
Rename `"parallax_toggles"` to `"toggles_secondary"`, keep the value of it tho here.

Reply:
Resolved.